internal/kuberesource/sets.go

```diff
@@ -709,7 +709,8 @@ func GPU() []any {
 func Vault(namespace string) []any {
 	vaultSfSets := StatefulSet("vault", "").WithAnnotations(map[string]string{
 		securePVAnnotationKey: "state:share",
-	}).
+	},
+	).
```

Comment:
Please revert this hunk - these should only be on different lines when they have different indentation level.